<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-V2MT19DS" failures="1" tests="2" name="testcases.SelectItemsFromMenuTest" time="47.306" errors="0" timestamp="2023-06-07T10:49:03 BDT" skipped="0">
  <testcase classname="testcases.SelectItemsFromMenuTest" name="iPhone14ProMaxCasesTest" time="12.971"/>
  <system-out/>
  <testcase classname="testcases.SelectItemsFromMenuTest" name="iPhone14ProMaxCasesTestImproved" time="34.335">
    <failure message="expected [https://gadgetandgear.com/category/iphone-14-pro-max] but found [https://gadgetandgear.com/]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [https://gadgetandgear.com/category/iphone-14-pro-max] but found [https://gadgetandgear.com/]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at testcases.SelectItemsFromMenuTest.iPhone14ProMaxCasesTestImproved(SelectItemsFromMenuTest.java:32)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- iPhone14ProMaxCasesTestImproved -->
  <system-out/>
</testsuite> <!-- testcases.SelectItemsFromMenuTest -->
